### PR TITLE
Add concerns and complaints

### PR DESCRIPTION
**Pull request by rgarner**
# [Add "Concerns and complaints" to the information sheet (static)](https://trello.com/c/Rvs9PGTM/279-2-add-concerns-and-complaints-to-the-information-sheet-static)

image

### CODE REVIEW

File: app/views/research_sessions/preview.html.erb [hunk 1]
```diff
@@ -114,6 +114,23 @@
...
   </section>
+  <section>
+    <h3 class="subtitle-small" id="private">Concerns and complaints</h3>
+
+    <p>If you have any concerns or a complaint, or would like to have your data deleted please contact:
+
+    <dl class="definition-list">
+      <div>
+        <dt class="definition-list__term">Telephone:</dt>
+        <dd>02084987869</dd>
```
Comment (tosbourn) [inline app/views/research_sessions/preview.html.erb]: Would it be worth making this a link with `href="tel:+442084987869"` to help folk browsing from outside the UK correctly contact the correct number.

Comment (joelcarr) [reply app/views/research_sessions/preview.html.erb]: As mentioned offline, this page is for printing so the functionality to call isn't needed here. However prepending the number with +44 should be done.

File: app/views/research_sessions/preview.html.erb [hunk 2]
```diff
@@ -114,6 +114,23 @@
       collected as part of the research is stored securely and kept strictly
       confidential.</p>
   </section>
+  <section>
+    <h3 class="subtitle-small" id="private">Concerns and complaints</h3>
```
Comment (joelcarr) [inline app/views/research_sessions/preview.html.erb]: Is there a reason for the ID?

Comment (rgarner) [reply app/views/research_sessions/preview.html.erb]: We meant to link to individual sections, which we aren't currently doing, but to keep with the convention I've updated it to `concerns`

**Pull request merged**